lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -0,0 +1,95 @@
+module Pod
```

Comment:
Missing tests for this class now added! Test integration with test targets and shell script phases!